extensions/servlet/test/com/google/inject/servlet/ServletDefinitionPathsTest.java

```diff
@@ -93,6 +93,12 @@ protected void service(HttpServletRequest servletRequest,
     expect(request.getServletPath())
         .andReturn(requestPath);
 
+    // computePath()
+    if(requestPath.isEmpty()) {
+        expect(request.getRequestURI()).andReturn(requestPath);
```

Comment:
style nits: please indent 2 spaces for new lines (4 spaces for continuations of prior lines)